elk/list.py

```diff
@@ -19,8 +19,11 @@ def list_runs(args):
     )
     for timestamp, run in subfolders:
         # Read the arguments used to run this experiment
-        with open(run / "args.json", "r") as f:
-            run_args = json.load(f)
+        try:
```

Comment:
Isn't it cleaner to check for the file with an if instead of using a try / except and then continue?

Reply:
The reason I did it this way is that `args.json` really _should_ exist, it's an anomaly if it doesn't exist. Arguably we should be indicating the error somewhere but `elk list` is going to change a lot soon anyway so I can't be bothered